python/sglang/srt/layers/quantization/quark/schemes/quark_w4a4_mxfp4.py

```diff
@@ -20,6 +21,63 @@
 
 OCP_MX_BLOCK_SIZE = 32
 
+def b_mxfp4_to_f32(x):
```

Comment:
consider to add comment to the 2 functions below, seems just one different.
consider to move `mxfp4_list` definition out of local.